github/ci/prow-deploy/files/jobs/kubevirt/hostpath-provisioner/hostpath-provisioner-presubmits.yaml

```diff
@@ -33,3 +33,65 @@ presubmits:
           resources:
             requests:
               memory: "29Gi"
+  - name: pull-hostpath-provisioner-sanity
+    skip_branches:
+      - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
+    always_run: false
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 1h
+      grace_period: 5m
+    max_concurrency: 6
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror: "true"
+    spec:
+      containers:
+        - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
+          command:
+            - "/usr/local/bin/runner.sh"
+            - "/bin/sh"
+            - "-c"
+            - "make test-sanity"
+          # docker-in-docker needs privileged mode
+          securityContext:
+            privileged: true
```

Comment:
Related to the dind question, if we don't need find we probably don't need privileged security context either.

Reply:
Unfortunately we need the privileged container, as the container I create needs to be privileged as well otherwise some of the tests won't have permission to bind mount which will fail the test.